script.js

```diff
@@ -0,0 +1,10 @@
+let userName = prompt("hello! what is your name?");
```

Comment:
Always use const for variables that won't change

```suggestion
const userName = prompt("hello! what is your name?");
```